lambdas/service/app.py

```diff
@@ -992,22 +992,26 @@ def repository_search(entity_type: str, item_id: Optional[str]) -> JSON:
 filters_param_spec = params.query(
     'filters',
     schema.optional(application_json(schema.object_type(
-        default='{}',
-        example={'cellCount': {'within': [[10000, 1000000000]]}},
-        properties={
+        {
             facet: {
                 'oneOf': [
                     schema.object(is_=schema.array({})),
                     *(
-                        schema.object_type({
-                            op: schema.array({}, minItems=2, maxItems=2)
+                        schema.object(**{
```

Comment:
Same question as before: why does this need to change from `object_type` to `object`?

Reply:
By changing this from `object_type` to`object` the properties for the relations `within`, `contains`, and `intersects` become more constraint by specifying a `required` field, as well as having `additionalProperties=False`. Without the 
addition of the `additionalProperties=False` invalid filters can be passed to the ES controller. see https://github.com/DataBiosphere/azul/pull/1849#discussion_r572266170

Because we use define sub-schemas with [oneof](https://json-schema.org/understanding-json-schema/reference/combining.html#oneof) we should be able to omit the `required` field (which is currently defined for the `is` relation).  This would allow us to keep `object_type` as long as we explicitly add `additionalProperties=False`  to all relations. 

At some point when `filters_param_spec` was added it this was changed from `object` to `object_type` within the same PR. Here is [1/3] that introduces `object` https://github.com/DataBiosphere/azul/commit/aadd9730de1b75df6697a7fe4d7667d947e37558 and [3/3] https://github.com/DataBiosphere/azul/commit/58738ae6010c52678681c1248537afb3918063f5 that changes it to `object_type`, I don't see any review comments that indicate why one was selected of the the other, or why only the `is` relation stayed with `object`. https://github.com/DataBiosphere/azul/pull/1647

We can either use `object` and keep the `required` field, or add the `additionalProperties=False` property to the `object_type`, and modify the `is` relation to match the other relations.